docs/design/minimax_m3_msa_support.html

```
@@ -0,0 +1,274 @@
+M3's MoE: how far its nearest neighbour sits in the <code>(topk, experts, hidden, inter)</code>
+normalized-log space. Error correlates with that distance at <strong>r = 0.85</strong> — but
+<strong>M3's nearest-neighbour distance (4.1) is beyond every validated case (max 2.5)</strong>. M3
+borrows <code>topk6/256/4096/2048</code> for <code>topk4/128/6144/3072</code> — and there is
+<strong>no same-topk neighbour in the pool at all</strong>.</p>
+<p class="note">So M3's MoE term is an <strong class="bad">out-of-support extrapolation</strong>: the
+linear fit reads ~88%, but that number is itself unvalidated — the honest statement is "no validation
+support; error likely ≥50%, unquantifiable." The earlier "~25–40%" was too optimistic. Why does the
+total decode TPOT still land at 2%? Because MSA (the other ~half) transfers cleanly and the MoE error
+happens to cancel at this operating point — <strong>not</strong> because the MoE term is individually
+accurate. <strong>The one real fix is to collect a <code>topk4 / 128-expert</code> MoE point</strong>
+(brings M3's neighbour distance into the validated <0.5 / ~5%-error range). Figures:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)